onadata/apps/logger/models/xform.py

```diff
@@ -89,6 +99,7 @@ class XForm(BaseModel):
 
     has_kpi_hooks = LazyDefaultBooleanField(default=False)
     kpi_asset_uid = models.CharField(max_length=32, null=True)
+    pending_delete = LazyDefaultBooleanField(default=False)
```

Comment:
Apparently we no longer need this lazy-field stuff starting with [PostgreSQL 11](https://www.postgresql.org/docs/11/sql-altertable.html) :tada: